.changeset/tasty-terms-remain.md

```diff
@@ -0,0 +1,5 @@
+---
+'xstate': minor
+---
+
+Added interop observable symbols to `ActorRef` and `Subscribable` so that actor refs are compatible with libraries like RxJS.
```

Comment:
As far as I can tell this has only been added to the `ActorRef` interface, right?
```suggestion
Added interop observable symbols to `ActorRef` so that actor refs are compatible with libraries like RxJS.
```

Reply:
Yes, was reverted later for `Subscribable`